src/main/java/com/powsybl/openloadflow/network/LfContingency.java

```diff
@@ -154,14 +151,10 @@ public void apply(LoadFlowParameters.BalanceType balanceType) {
             PowerShift shift = e.getValue();
             bus.setLoadTargetP(bus.getLoadTargetP() - getUpdatedLoadP0(bus, balanceType, shift.getActive(), shift.getVariableActive()));
             bus.setLoadTargetQ(bus.getLoadTargetQ() - shift.getReactive());
-            Set<String> loadsIdsInContingency = originalPowerShiftIds.stream()
-                    .distinct()
-                    .filter(bus.getLoad().getOriginalIds()::contains) // maybe not optimized.
-                    .collect(Collectors.toSet());
-            if (!loadsIdsInContingency.isEmpty()) { // it could be a LCC in contingency.
-                bus.getLoad().setAbsVariableTargetP(bus.getLoad().getAbsVariableTargetP() - Math.abs(shift.getVariableActive()));
-                loadsIdsInContingency.stream().forEach(loadId -> bus.getLoad().setOriginalLoadDisabled(loadId, true));
-            }
+            bus.getLoad().ifPresent(load -> { // it could be a LCC in contingency.
```

Comment:
This is not equivalent at all... imagine in a bus we have 2 `Load` d1 and d2 and a LCC converter station. If the contingency only concerns d1, only d1 must be disabled through `load.setOriginalLoadDisabled(loadId, true)`. If the contingency concerns the LCC converter station, nothing must be done on the `LfLoad`.

Reply:
Fixed